src/bokeh/command/subcommands/settings.py

```diff
@@ -0,0 +1,197 @@
+#-----------------------------------------------------------------------------
+# Imports
+#-----------------------------------------------------------------------------
+
+# Standard library imports
+from argparse import Namespace
+from typing import Any
+
+# Bokeh imports
+from bokeh.settings import PrioritizedSetting, _Unset, settings
+from bokeh.util.settings import get_all_settings
+
+# Bokeh imports
+from ..subcommand import Argument, Subcommand
+
+#-----------------------------------------------------------------------------
+# Globals and constants
+#-----------------------------------------------------------------------------
+
+__all__ = (
+    'Settings',
+)
+
+#-----------------------------------------------------------------------------
+# General API
+#-----------------------------------------------------------------------------
+
+class Settings(Subcommand):
+    ''' Subcommand to print information about Bokeh settings.
+
+    '''
+
+    name = "settings"
+
+    help = "Print information about Bokeh settings and their current values"
+
+    args = (
+
+        (('-v', '--verbose'), Argument(
+            action="store_true",
+            help="Show detailed help for a specific setting",
+        )),
+
+        ('setting_name', Argument(
+            nargs='?',
+            help="Name of a specific setting to show detailed help for (use with -v)",
+        )),
+
+    )
+
+    def invoke(self, args: Namespace) -> None:
+        '''
+
+        '''
+        all_settings = get_all_settings()
+
+        if not all_settings:
+            print("No settings found")
+            return
+
+        if args.setting_name:
+            if args.setting_name in all_settings:
+                if args.verbose:
+                    self._print_setting_detail(args.setting_name, all_settings[args.setting_name])
+                else:
+                    print("To get detailed help for a specific setting, use:")
+                    print("  bokeh settings [-v | --verbose] <setting_name>")
+                    print("\nFor a list of all settings, use:")
+                    print("  bokeh settings")
+            else:
+                print(f"Setting '{args.setting_name}' not found.")
+                print("Available settings:")
+                for name in sorted(all_settings):
+                    print(f"  {name}")
+        else:
+            self._print_settings_table(all_settings)
+
+    def _print_settings_table(self, all_settings: dict[str, PrioritizedSetting[Any]]) -> None:
+        ''' Print all settings in a table format.
+        '''
+        print("Bokeh Settings:")
+        print("=" * 80)
+        print(f"{'Setting':<30} {'Environment Variable':<35} {'Value':<25}")
+        print("-" * 80)
+
+        for name, attr in all_settings.items():
+            current_value = getattr(settings, name)()
+            env_var = attr.env_var
+            value_str = str(current_value)
+
+            print(f"{name:<30} {env_var:<35} {value_str:<25}")
```

Comment:
I think this can be simplified, please check: 
```suggestion
        for name, descriptor in all_settings.items():
            print(f"{name:<30} {attr.env_var:<35} {descriptor():<25}")
```

As I mentioned, there should not be a need to use `getattr` on `settings`, the descriptor object can just be called like a function to retrieve its value.  Also the f-string will implicitly do a plain `str` conversion, no need to do that manually.